examples/shadow-amp/test.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
can we rename this file into something a bit more descriptive? (not sure what it is, but it feels like an example of some app init/bootstrapping code?)

Reply:
It's the app code for the test subject page

Reply:
Given the port below and stuff, maybe it's just better to be in the integration js file itself as a string?